src/plots/cartesian/layout_attributes.js

```diff
@@ -448,6 +448,18 @@ module.exports = {
             '*%H~%M~%S.%2f* would display *09~15~23.46*'
         ].join(' ')
     },
+    tickformatstops: {
+        valType: 'any',
```

Comment:
We can do better here.

See for example the `layout.slider` declaration [here](https://github.com/plotly/plotly.js/blob/ff8ecb1b6a63559a897c2ba744cecdef0c6660a8/src/components/sliders/attributes.js#L34). The `steps` array container is declared using the :sparkles: special :sparkles: `_isLinkedToArray`. 

So, this here should look like:

```js
tickformatstops: {
  _isLinkedToArray: 'tickformatstop',

  dtickrange: {
    valType: 'info_array',
    // ... declared just like 'range' above
  },
  values: {
     valType: 'any', 
     // ...
  }
}